### DISCUSSION TITLE
Get entityid of dialog form in propertygrid template

### DISCUSSION THREAD

**Discussion by rhemasystems** [serenity-is/Serenity#6910] [Help] [answered]
Please I have an entitydialog inside a tab of a master dialog
When the master dialog is opened, it loads this entity dialog in the other tab, this form can either be empty or have a record already saved in the db

![image](https://github.com/serenity-is/Serenity/assets/50826372/aaf82c53-061e-4f67-925a-5da3bc1da4ef)

How do I get the circled  Id value which is the entityid of this other form shown below to know whether to call the insert service if its null or the update service if it has a value , after the  created **Save** button is clicked.

I used the **OtherFormInTab** sample to create this

![image](https://github.com/serenity-is/Serenity/assets/50826372/be87e01b-17fa-43e2-b0c0-9b5b84418dc4)

Master Dialog constructor
![image](https://github.com/serenity-is/Serenity/assets/50826372/990002bf-6ea0-4cf8-b3e4-21552d1607dc)

**Comment by rhemasystems** [comment]
Please any assistance will be much appreciated. Thank you

**Reply by luongnv6** [reply, marked as answer]
```ts
var thisform = this;
var id = thisform.churchlifeform.Id.value;
```
